programs/whirlpool/src/state/adaptive_fee_tier.rs

```diff
@@ -149,6 +149,22 @@ impl AdaptiveFeeTier {
     }
 }
 
+#[cfg(test)]
+mod discriminator_tests {
+    use anchor_lang::Discriminator;
+
+    use super::*;
+
+    #[test]
+    fn test_discriminator() {
+        let discriminator = AdaptiveFeeTier::discriminator();
+        // The discriminator is determined by the struct name and not depending on the program id.
+        // $ echo -n account:AdaptiveFeeTier | sha256sum | cut -c 1-16
+        // 931090742f92952e
+        assert_eq!(discriminator, [147, 16, 144, 116, 47, 146, 149, 46]);
```

Comment:
nice code comment!

Then what do you think about `[0x93, 0x10, 0x90, 0x74, 0x2f, 0x92, 0x95, 0x2e]` instead of `[147, 16, 144, 116, 47, 146, 149, 46]` ?
It is easy to verify to me.
Do top-tier programmers instantly convert hex to decimal just by looking at it? 😆